Source: @pullmoll
Upstream: no
Reason: Newer binutils have slightly changed macros for bdf.

--- libutil++/bfd_support.cpp	2015-07-07 16:04:37.000000000 +0200
+++ libutil++/bfd_support.cpp	2020-05-27 20:32:36.305059578 +0200
@@ -137,7 +137,7 @@
 		}
 	}
 
-	bfd_size_type buildid_sect_size = bfd_section_size(ibfd, sect);
+	bfd_size_type buildid_sect_size = bfd_section_size(sect);
 	char * contents = (char *) xmalloc(buildid_sect_size);
 	errno = 0;
 	if (!bfd_get_section_contents(ibfd, sect,
@@ -188,7 +188,7 @@
 	if (sect == NULL)
 		return false;
 	
-	bfd_size_type debuglink_size = bfd_section_size(ibfd, sect);  
+	bfd_size_type debuglink_size = bfd_section_size(sect);  
 	char * contents = (char *) xmalloc(debuglink_size);
 	cverb << vbfd
 	      << ".gnu_debuglink section has size " << debuglink_size << endl;
@@ -346,7 +346,7 @@
 	// first restrict the search on a sensible range of vma, 16 is
 	// an intuitive value based on epilog code look
 	size_t max_search = 16;
-	size_t section_size = bfd_section_size(abfd, section);
+	size_t section_size = bfd_section_size(section);
 	if (pc + max_search > section_size)
 		max_search = section_size - pc;
 
@@ -819,10 +819,10 @@
 	else
 		pc = (sym.value() + offset) - sym.filepos();
 
-	if ((bfd_get_section_flags(abfd, section) & SEC_ALLOC) == 0)
+	if ((bfd_section_flags(section) & SEC_ALLOC) == 0)
 		goto fail;
 
-	if (pc >= bfd_section_size(abfd, section))
+	if (pc >= bfd_section_size(section))
 		goto fail;
 
 	ret = bfd_find_nearest_line(abfd, section, syms, pc, &cfilename,
--- opjitconv/create_bfd.c	2015-06-05 22:39:34.000000000 +0200
+++ opjitconv/create_bfd.c	2020-05-27 20:38:40.445045059 +0200
@@ -86,12 +86,12 @@
 		bfd_perror("bfd_make_section");
 		goto error;
 	}
-	bfd_set_section_vma(abfd, section, vma);
-	if (bfd_set_section_size(abfd, section, size) == FALSE) {
+	bfd_set_section_vma(section, vma);
+	if (bfd_set_section_size(section, size) == FALSE) {
 		bfd_perror("bfd_set_section_size");
 		goto error;
 	}
-	if (bfd_set_section_flags(abfd, section, flags) == FALSE) {
+	if (bfd_set_section_flags(section, flags) == FALSE) {
 		bfd_perror("bfd_set_section_flags");
 		goto error;
 	}
